lxd/storage/drivers/driver_btrfs_volumes.go

```diff
@@ -455,10 +455,16 @@ func (d *btrfs) CreateVolumeFromCopy(vol Volume, srcVol Volume, copySnapshots bo
 
 	var snapshots []string
 
+	// Check if there is at least one snapshot that requires copying.
+	copySnapshots := false
+	if len(vol.Snapshots) > 0 {
+		copySnapshots = true
+	}
+
 	// Get snapshot list if copying snapshots.
 	if copySnapshots && !srcVol.IsSnapshot() {
```

Comment:
We can lose copySnapshots now and just have `if len(vol.Snapshots) > 0 && !srcVol.IsSnapshot() {` right?

Reply:
Yes, thanks! Much simpler this way.